test/test.track.js

```diff
@@ -92,3 +92,20 @@ tap.test('can use /t.gif route to get a tracking pixel', (t) => {
     });
   });
 });
+
+tap.test('can use /c.gif route to get a conversion tracking pixel', (t) => {
+  setup.server.inject({
+    url: '/c.gif?type=thisType',
```

Comment:
lets make sure we are passing similar payload as the conversion api call https://github.com/firstandthird/micro-metrics/pull/70/files#diff-28130ac63d341780461092a6591f2552R52